### DISCUSSION TITLE
Roman Numeral List

### DISCUSSION THREAD

**Discussion by calvinkattkyndryl** [community/community#136331] [Programming Help]
Is there any way like the numbered list to create roman Numeral list such as
(I)
(II)
(III)
(IV)

**Comment by ebndev** [comment]
Hi @calvinkattkyndryl, thanks for being a part of the GitHub Community!

If you're using HTML, you can add this attribute to your list element:

```
<ol type="I">
  <li>first item</li>
  <li>second item</li>
</ol>
```

**Reply by dannysigalovich** [reply]
You can use CSS , can try this 'list-style-type' property:
```
<ul style="list-style-type: upper-roman;">
  <li>First item</li>
  <li>Second item</li>
  <li>Third item</li>
  <li>Fourth item</li>
</ul>
```
